src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,33 +1,43 @@
 package edu.kis.vh.nursery;
 
+/*
+ * Wyliczanki zostaly zachowane, aby mozna bylo korzystac z konkretnych implementacji wyliczanek
+ */
 public class DefaultCountingOutRhymer {
 
-	private int[] NUMBERS = new int[12];
+	static final int EMPTY = -1;
+	static final int NUMBER_CAPACITY = 12;
+	static final int FULL = 11;

```

Comment:
This constant is related to NUMBER_CAPACITY . Changing only one leads to an error (inconsistency).